crates/bevy_ecs/src/storage/table.rs

```diff
@@ -48,12 +47,25 @@ impl Column {
         }
     }
 
+    #[inline]
+    fn ticks_mut(&mut self) -> &mut Vec<ComponentTicks> {
+        // SAFE: unique access
+        unsafe { &mut *self.ticks.get() }
```

Comment:
We can make this safe using `self.ticks.get_mut()`

Reply:
Oh cool! I for some reason was unable to find this method. I see it's stabilised since 1.50.
Note that this UnsafeCell goes away in #2227 completely.